### PR TITLE
TW-1437: Remove inactive and irrelevant buttons

### PR DESCRIPTION
**Pull request by nqhhdev**
### Ticket 

- #1437
- #1082
- #1420 
- #1412 
- #1418 

### Resolved 

- [x] Remove Share location and share contact button

Screenshot 2024-02-01 at 10 03 54

- [x] Remove Chat Folders

Screenshot 2024-02-01 at 14 40 13

- [x] Remove Emote settings
Screenshot 2024-02-01 at 10 03 39

- [x] Update background color

Screenshot 2024-02-01 at 10 04 43

Screenshot 2024-02-01 at 10 05 27

- [x] Remove More button when select chat in chat list

Screenshot 2024-02-01 at 10 09 50

- [x] Remove extra text from Profile information

Screenshot 2024-02-01 at 10 17 33

- [x] Change naming in other users' profile info

Screenshot 2024-02-01 at 14 38 32

- [x] Update language screen

Screenshot 2024-02-01 at 14 37 47

Screenshot 2024-02-01 at 14 37 57

- [x] Update button navigator bar  

Screenshot 2024-02-01 at 14 38 16

### CODE REVIEW

**Comment by hoangdat** [comment]
- [ ] background color of bottom navigator bar
- [ ] update size for bottom navigator bar

**Review by sherlockvn — APPROVED** [review]

File: lib/presentation/enum/settings/settings_enum.dart [hunk 1]
```diff
@@ -41,7 +41,7 @@ enum SettingEnum {
   String subtitleSettings(BuildContext context) {
     switch (this) {
       case SettingEnum.chatSettings:
-        return L10n.of(context)!.settingsChatSubtitle;
+        return '';
```
Comment (imGok) [inline lib/presentation/enum/settings/settings_enum.dart:44]: why we don't delete SettingEnum.chatSettings if we don't use it anymore

File: lib/utils/string_color.dart [hunk 2]
```diff
@@ -36,4 +36,8 @@ extension StringColor on String {
     _colorCache[this] ??= {};
     return _colorCache[this]![0.5] ??= _getColorLight(0.5);
   }
+
+  String capitalize() {
```
Comment (imGok) [inline lib/utils/string_color.dart:40]: should be in `string_extension.dart`

File: lib/presentation/enum/settings/settings_enum.dart (hunk 1 above)
Comment (nqhhdev) [reply lib/presentation/enum/settings/settings_enum.dart:44]: I don't think it's necessary, it's okay to update the value later

File: lib/utils/string_color.dart (hunk 2 above)
Comment (nqhhdev) [reply lib/utils/string_color.dart]: updated

File: assets/l10n/intl_en.arb [hunk 3]
```diff
@@ -2969,7 +2969,7 @@
   "groupName": "Group name",
   "descriptionHelper": "You can provide an optional description for your group.",
   "groupNameCannotBeEmpty": "Group name cannot be empty",
-  "sharedMediaAndFiles": "Shared media and files",
+  "sharedMediaAndLinks": "Shared media and links",
```
Comment (hoangdat) [inline assets/l10n/intl_en.arb:2972]: if only change it, other language still not update too.
Solution: delete the old one for all `intl_**` file

File: lib/pages/settings_dashboard/settings_profile/settings_profile_view_web.dart [hunk 4]
```diff
@@ -255,16 +255,6 @@ class SettingsProfileViewWeb extends StatelessWidget {
                   ],
                 ),
               ),
-              Padding(
-                padding:
-                    SettingsProfileViewWebStyle.paddingWidgetEditProfileInfo,
-                child: Text(
-                  L10n.of(context)!.editWorkIdentitiesDescriptions,
```
Comment (hoangdat) [inline lib/pages/settings_dashboard/settings_profile/settings_profile_view_web.dart:262]: please delete this text also in all `intl_**` files

**Pull request merged**